lib/manageiq/ui/classic/js_compressor.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ
+  module UI
+    module Classic
+      class JsCompressor < Sprockets::UglifierCompressor
+        def initialize(options = {})
```

Comment:
Note, this interface is changed between 3.7.3 and 4.2.1

https://github.com/rails/sprockets/blob/v3.7.3/lib/sprockets/uglifier_compressor.rb#L37

https://github.com/rails/sprockets/blob/v4.2.1/lib/sprockets/uglifier_compressor.rb#L39